lib/systems/parse.nix

```diff
@@ -71,7 +71,7 @@ rec {
   kernels = with execFormats; with kernelFamilies; setTypesAssert "kernel"
     (x: isExecFormat x.execFormat && all isKernelFamily (attrValues x.families))
   {
-    cygwin  = { execFormat = pe;      families = { inherit /*unix*/ windows-nt; }; };
+    cygwin  = { execFormat = pe;      families = { inherit unix windows-nt; }; };
```

Comment:
Could also add cygwin support per-package, but it seems to be that cygwin is as unixy as the others here.